lib/active_merchant/billing/gateways/payu_latam.rb

```diff
@@ -152,7 +152,7 @@ def add_order(post, options)
         order[:accountId] = @options[:account_id]
         order[:partnerId] = options[:partner_id] if options[:partner_id]
         order[:referenceCode] = options[:order_id] || generate_unique_id
-        order[:description] = options[:description] || 'unspecified'
+        order[:description] = options[:description] || 'Compra en ' + @options[:merchant_id]
```

Comment:
Just to sanity check here, it makes to add `Compra en` before the user-supplied string? I trust you, just it's the only instance of this I could find in AM (not that I spent _that_ long looking).

Reply:
Yea from the ticket: 

"PayU Latam requires their description field. We noted in our documentation that it is a required GSF, however, if it's not sent, then we currently send "unspecified" and PayULatam wants us to default to "Compra en insert merchant name". We can't do this because we don't have the merchant name. But we do have their merchant_id from their gateway credentials which they said are approved."

Reply:
This is a weird gateway.